### PR TITLE
NEG: Add wrappers for NEG Windows test.

### PR DESCRIPTION
**Pull request by shettyg**
With this commit, you can run the NEG tests for Windows with: -test.run=TestWindowsNEG.

### CODE REVIEW

File: cmd/e2e-test/neg_test.go [hunk 1]
```diff
@@ -138,6 +146,14 @@ func TestNEG(t *testing.T) {
 }
 
 func TestNEGTransition(t *testing.T) {
+	testNEGTransitionOS(t, e2e.Linux)
+}
+
+func TestWindowsNEGTransition(t *testing.T) {
```
Comment (aojea) [inline cmd/e2e-test/neg_test.go:152]: naive question, how do you skip this test if there are no windows nodes?

or is ok to fail in that case?

Comment (shettyg) [reply cmd/e2e-test/neg_test.go:152]: It is okay to fail in that case.

**Review by swetharepakula — COMMENTED** [review]
Thanks for the change!

/lgtm
/approve

**Pull request merged**